From: Thadeu Lima de Souza Cascardo <cascardo@debian.org>
Date: Fri, 30 Apr 2021 12:58:26 -0700
Subject: Build targets shoult not attempt network access

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=898707
Origin: vendor, 
Forwarded: not-needed
Last-Update: 2018-05-17

When git is installed, the eppic extensions makefile will try to access
github.com to clone the extension code.
 This patch simply fails to find the git binary by adding an extra
 false condition to minimize the delta size.
---
 extensions/eppic.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extensions/eppic.mk b/extensions/eppic.mk
index bda69da..66027a0 100644
--- a/extensions/eppic.mk
+++ b/extensions/eppic.mk
@@ -32,7 +32,7 @@ all:
         then \
         if  [ ! -f $(APPFILE) ]; \
         then \
-          if [ -f "$(GIT)" ]; \
+          if [ -f "$(GIT)" -a 0 -gt 1 ]; \
           then \
              if [ -n "$(EPPIC_GIT_URL)" ]; then \
                git clone "$(EPPIC_GIT_URL)" eppic; \
